python/ray/function_manager.py

```diff
@@ -641,12 +635,9 @@ def temporary_actor_method(*xs):
             traceback_str = ray.utils.format_error_message(
                 traceback.format_exc())
             # Log the error message.
-            push_error_to_driver(
-                self._worker,
-                ray_constants.REGISTER_ACTOR_PUSH_ERROR,
-                traceback_str,
-                driver_id,
-                data={"actor_id": actor_id.binary()})
+            push_error_to_driver(self._worker,
+                                 ray_constants.REGISTER_ACTOR_PUSH_ERROR,
+                                 traceback_str, driver_id)
```

Comment:
Let's replace `traceback_str` with

`"Failed to unpickle actor class '{}' for actor ID {}. Traceback:\n{}".format(class_name, actor_id.hex(), traceback_str)`

and make sure it looks reasonable?